Project4/index.html

```diff
@@ -0,0 +1,257 @@
+    <link rel="stylesheet" href="style.css">
+    <link href="//netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Gugi|Khand|Orbitron|Teko" rel="stylesheet">
+
+    <title>Project4-MVTA</title>
+</head>
+
+<body>
+    <main>
+        <h1>MVTA
+        </h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="#register">register</a>
+                </li>
+                <li>
+                    <a href="#speaker">speaker</a>
+                </li>
+                <li>
+                    <a href="#schedule">schedule</a>
+                </li>
+                <li>
+                    <a href="#sponser">sponser</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="overlay-item overlay-effect">
+            <img class="conference-image" src="images/photomotion.jpg" alt="motion photo">
+            <!-- https://www.lifewire.com/5g-speed-4180992 -->
+            <div class="overlay">
+                <div class="mask">
+                    <h3>Motion Video Technology and Art</h3>
+                    <p>
+                        Motion Video Technology and Art (MVTA) for people in Motion Graphic video artists and
+                        developers.
+                        Each year the conference invites experts specializing in this particular area to speakers
+                        about
+                        their projects and what their jobs specifically entail, and a Q&A time with the audience.
+                    </p>
+                    <p>
+                        May 13-15
+                    </p>
+                    <p>
+                        Sanfransico ,Califonia
+                    </p>
+                </div>
+            </div>
+        </div>
+        <button class="mt medium diagonal">Ticket</button>
+        </div>
+        <footer>
+            <h3 id="register">Register</h3>
+            <form class="register">
+                <div class="register-switch">
+                    <input type="radio" name="sex" value="F" id="sex_f" class="register-switch-input" checked>
+                    <label for="sex_f" class="register-switch-label">Female</label>
+                    <input type="radio" name="sex" value="M" id="sex_m" class="register-switch-input">
+                    <label for="sex_m" class="register-switch-label">Male</label>
+                </div>
+                <input type="email" class="register-input" placeholder="Email address">
+                <input type="password" class="register-input" placeholder="Password">
+                <input type="submit" value="Create Account" class="register-button">
+            </form>
+
+            <h3 id="speaker">Speakers
+            </h3>
+            <div class="speaker-details">
+                <img src="images/jean-babtiste kempf photo.jpg " alt="photo of jean-babtiste kempf" width="200"
```

Comment:
1. Your image filename should not include spaces. This will work on some web servers, but not all, so should be avoided.
1. Width and height should be controlled via CSS vs inline attributes on `img`.

```suggestion
                <img src="images/jean-babtiste kempf photo.jpg " alt="photo of jean-babtiste kempf">
```

This is true for all your speaker details, but I'll only comment on it here.